src/cache/cache_las.c

```diff
@@ -1237,9 +1238,21 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
         /*
          * Remove the previous obsolete entries whenever the scan moves into next key or the current
          * LAS record is not a modify record.
+         *
+         * The special ondisk value i.e both timestamp and transaction id values as 0 and is the
```

Comment:
Is a "special" update one that is created by `__rec_append_orig_value`?

Reply:
yes